### PR TITLE
Initial draft of shutdown.rst against the new GUI

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: userguide/shutdown.rst [hunk 1]
```diff
@@ -18,15 +17,19 @@ will negatively impact current users of the %brand% system.
...
 
 
-If a scrub or resilver is in progress when a shutdown is requested, an
-additional warning will ask if you wish to proceed. In this case, it
-is recommended to :guilabel:`Cancel` the shutdown request and to
-periodically run :command:`zpool status` from :ref:`Shell` until the
-scrub or resilver process is complete. Once complete, the shutdown
-request can be re-issued.
+.. If a scrub or resilver is in progress when a shutdown is requested, an
+   additional warning will ask if you wish to proceed. In this case, it
```
Comment (wblock) [inline userguide/shutdown.rst]: s/will ask/asks/

Avoid "you": "whether it should proceed."

File: userguide/shutdown.rst [hunk 2]
```diff
@@ -18,15 +17,19 @@ will negatively impact current users of the %brand% system.
...
+   is recommended to :guilabel:`Cancel` the shutdown request and to
```
Comment (wblock) [inline userguide/shutdown.rst]: This is confusing and complicated.  Let's just say "If a scrub or resilver is running, a warning is shown. Clicking :guilabel:`Cancel` is recommended. :command:`zpool status` can be run from the :ref:`Shell` to watch for the scrub or resilver to complete. Then the system can be shut down normally."

File: userguide/shutdown.rst [hunk 3]
```diff
@@ -18,15 +17,19 @@ will negatively impact current users of the %brand% system.
...
+   periodically run :command:`zpool status` from :ref:`Shell` until the
+   scrub or resilver process is complete. Once complete, the shutdown
+   request can be re-issued.
+
+   ^commented out because was unable to test this. Scrubs were
```
Comment (wblock) [inline userguide/shutdown.rst]: Did you build-test this?

File: userguide/shutdown.rst [hunk 4]
```diff
@@ -18,15 +17,19 @@ will negatively impact current users of the %brand% system.
...
+   completing very quickly and couldn't shut it down while it
+   was scrubbing postponed until later date.
 
 Click the :guilabel:`Cancel` button to cancel the shutdown request.
-Otherwise, click the :guilabel:`Shutdown` button to halt the system.
-Shutting down the system disconnects all clients, including the web
-administration GUI, and powers off the %brand% system. Physical access
-to the %brand% system will be needed to turn it back on.
+Check the :guilabel:`Confirm` check box and click the
+:guilabel:`Shutdown` button to shutdown the system. Shutting down the
```
Comment (wblock) [inline userguide/shutdown.rst]: I don't think we actually need to tell the user that Cancel means cancel.

File: userguide/shutdown.rst [hunk 5]
```diff
@@ -18,15 +17,19 @@ will negatively impact current users of the %brand% system.
...
+system disconnects all clients, including the web administration GUI.
+Physical access to the %brand% system is needed to turn it back on.
```
Comment (wblock) [inline userguide/shutdown.rst]: s/needed/required/

**Comment by dlavigne** [comment]
Aaron: please also create a PR on master for the text improvements.

**Pull request merged**